src/test/java/com/rakovets/course/java/core/practice/lambda_expressions/TvTest.java

```diff
@@ -0,0 +1,113 @@
+package com.rakovets.course.java.core.practice.lambda_expressions;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TvTest {
+    static List<TV> tv = Arrays.asList(
+            new TV("Sony", "1A", 2021, 43, 800.0),
+            new TV("Sony", "2A", 2020, 23, 150.0),
+            new TV("LG", "8H", 2021, 43, 600.0),
+            new TV("LG", "7Н", 2020, 34, 400.0),
+            new TV("LG", "2Н", 2018, 23, 145.0),
+            new TV("Samsung", "3CC", 2021, 43, 1250.0),
+            new TV("Panasonic", "4DD", 2020, 52, 1250.0),
+            new TV("Philips", "2EE", 2021, 43, 700.0));
+
+    @Test
+    static Stream<Arguments> specificDiagonalTestProviderArguments() {
+        return Stream.of(
+                Arguments.of(Arrays.asList(tv.get(1), tv.get(4)), 23),
+                Arguments.of(Arrays.asList(tv.get(3)), 34)
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("specificDiagonalTestProviderArguments")
+    public void specificDiagonalTest(List<TV> expected, int diagonal) {
+        List<TV> actual = TvFilter.specificDiagonal(tv, diagonal);
+        assertEquals(expected, actual);
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"